### ISSUE TITLE
Add names to drives in folder field (Windows)

### ISSUE THREAD

**Issue by rebio** [Beep6581/RawTherapee#6157]
TLDR: Add names to drives in the folder field on the file browsing page. So far only the favourites section has named drives.

So far there is just the drive letter in the HDD structure/folder field. I named my drives and don't really recognize them just by their drive letter. Some of the HDDs/folders I want to use are not listed in the favourites section.
Example: When I plug in my USB-card reader many _fake drives_ show up. In the Windows browser the one with the images has a name and the others are blank. Because Windows is Windows the USB card reader sometimes has a different drive letters assigned to the single card reader slots. It does not matter if they are in use or not.

My suggestion is to add the names to drives so that it's easier to pick the correct drive.

![Suggestion - many drive names](https://user-images.githubusercontent.com/10911084/110108348-5d69e500-7dac-11eb-98f9-6e6b6655e20a.png)
